databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -188,7 +197,7 @@ def delete_share_cli(api_client, name):
 @click.option('--allowed_ip_address', default=None, required=False, multiple=True,
```

Comment:
@wchau Could you change this to use dashes to be consistent with everything else and check if other commands use the same? We shouldn't end up in a situation where folks depend on it.

Reply:
Done